### ISSUE TITLE
Death in Durotar, Spawned at Gadgetzan Graveyard

### ISSUE THREAD

**Issue by Cosmic-Cocoa** [chromiecraft/chromiecraft#30]
##### ISSUE: FACTION SIDE:

![image](https://user-images.githubusercontent.com/1884642/108204991-63a96180-711c-11eb-9ed9-138c233070a6.png)

##### SMALL DESCRIPTION:
I died fighting the Burning Blade Cultists in Dustwind Cave in Durotar. My ghost spawned at the Graveyard by Gadgetzan.

Here's a screenshot of my world map that shows my current location (as a ghost) as well as my corpse location:

![Durotar Death, Tanaris Graveyard](https://user-images.githubusercontent.com/78349628/106441627-e86b5e80-643f-11eb-9276-0723ee7cc9da.jpg)

##### EXPECTED BLIZZLIKE BEHAVIOUR:
My ghost should spawn at the graveyard in Durotar.

##### CURRENT BEHAVIOUR:
I died fighting the Burning Blade Cultists in Dustwind Cave in Durotar. My ghost spawned at the Graveyard by Gadgetzan.

##### STEPS TO REPRODUCE THE PROBLEM:

1. Try dying in Dustwind Cave? I was roughly in the middle of the cave when I died.

##### AC HASH/COMMIT:

https://github.com/chromiecraft/azerothcore-wotlk/commit/df600f9946c

##### OPERATING SYSTEM:

Ubuntu 20.04

##### MODULES:

- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)

##### OTHER CUSTOMIZATIONS:

None.

##### SERVER:

ChromieCraft

**Comment by Helias** [comment]
Were you alliance or horde?

**Comment by Cosmic-Lattee** [comment]
She is a horde troll shaman [Level 12] (I'm her husband). She was doing her shaman quest Call of Fire at the time.

I tried to replicate this with a level 1 troll named Deadboi. I died several times in a few different places in the cave and each time I respawned in the correct graveyard.

So I am guessing that it is either a very specific place within the caves or just some sort of strange combination that lead to her respawning over there.

**Comment by Helias** [comment]
Ok, thank you both, we need to test in deep for this, when we will be able to reproduce the bug we will fix it for sure 😉

**Comment by Hacaw** [comment]
**TL;DR** I reproduced it, after 15 dying attempts.

@Helias I cannot pin-point the exact x,y,z coords since I am not sure how to extract it from my current position, maybe someone can help me out. 

It's in the middle of the cave, you can check the mini-map. 
Also, the hint that made me discover it, was the channel-switch.

![image](https://user-images.githubusercontent.com/6963500/106511780-60d22e00-64d9-11eb-8549-035b49409096.png)


##### STEPS TO REPRODUCE THE PROBLEM:
1) .tele DustwindCave
2) go to half of the cave 

##### AC HASH/COMMIT: 
adcf3f7a

##### DATABASE VERSION: 
WORLD: 2021_02_01_00
AUTH: 2021_01_25_00
CHARACTERS: 2021_01_25_00

##### OPERATING SYSTEM:
Windows 10 - Docker for Desktop

##### EXTRA NOTES: 
Chromie-Client (Downloaded from 10-01-2021)
DBC & Maps - v9 - 1-12-2020

##### MODULES:
None

##### OTHER CUSTOMIZATIONS:
None

**Comment by Helias** [comment]
Hey, thank you a lot! I think that the screenshot with the minimap is enough to understand the exactly location

**Comment by LukasVolgger** [comment]
I would say that this is a feature. lol

**Comment by Stifler82** [comment]
> **TL;DR** I reproduced it, after 15 dying attempts.
> 
> @Helias I cannot pin-point the exact x,y,z coords since I am not sure how to extract it from my current position, maybe someone can help me out.

Just tag yourself and do the **.gps** command in chat

**Comment by Stifler82** [comment]
**I'm not sure, but I think it's something related to DBC or maps, mmaps, vmaps, although, I assume the problem is in the second case.**

https://user-images.githubusercontent.com/1884642/107545120-d49fc500-6bc2-11eb-88fd-d22a15bb3d14.mp4

**Comment by illfated** [comment]
Known issue in other repositories as well. Most likely a maps/vmaps issue:
**[3.3.5] On entering cave in durotar you can chat in tanaris** https://github.com/TrinityCore/TrinityCore/issues/19761

**Comment by Stifler82** [comment]
https://github.com/chromiecraft/chromiecraft/issues/303
